src/main/java/servlets/VendorServlet.java

```diff
@@ -49,24 +49,24 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(configMap);
   }
 
-  /** Delete the desired Vendor from the fileset and spreadsheets if posible.*/
+  /** Delete the desired Vendor from the fileset and spreadsheets.*/
   public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {  
     String vendorID = request.getParameter(VENDOR_ID_PARAM);
-    String deleteStatus = "success";
+    String responseString = "success"; 
 
     // Delete the file from the filesystem and update the sheets.
     try {
       deleteVendorFile(vendorID);
       updateSheets(request);    
-    } catch (FileNotFoundException e) {
-      deleteStatus = "Could not execute, an FileNotFoundException was thrown.";
+    } catch (IOException e) {
```

Comment:
If you handle the two types of exceptions identically, then they don't need to separate catch blocks.